code/modules/clothing/masks/boxing.dm

```diff
@@ -33,14 +31,13 @@
 	ADD_TRAIT(user, TRAIT_DIAGNOSTIC_HUD, MASK_TRAIT)
 	var/datum/atom_hud/H = GLOB.huds[DATA_HUD_DIAGNOSTIC_BASIC]
 	H.add_hud_to(user)
-	voice_unknown = TRUE
+	ADD_TRAIT(src, TRAIT_HIDES_VOICE, REF(src))
 
 /obj/item/clothing/mask/infiltrator/dropped(mob/living/carbon/human/user)
 	to_chat(user, "You pull off the balaclava, and the mask's internal hud system switches off quietly.")
-	REMOVE_TRAIT(user, TRAIT_DIAGNOSTIC_HUD, MASK_TRAIT)
```

Comment:
The diagnostic hud trait is still added, is this intentional?

Reply:
Nope good catch